### ISSUE TITLE
fortls doesn't work when ssh into linux

### ISSUE THREAD

**Issue by pjuangph** [hansec/vscode-fortran-ls#35]
I remoted into my linux environment and tried to set the path to fortls executable. It doesn't seem to recognize .F90 files. The same file works fine in windows.  I tried setting fortls path for the user/remote/workspace settings and vs code still doesn't recognize the file. 

This might be a problem with vs code. I don't know. 

I also wanted to add the file extension .cuf into fortran language server. This the cuda for fortran. I am writing a tutorial on how to get started with nvfortran (nvidia fortran compiler) https://github.com/pjuangph/fortran everything so far works except for the vs code extension.

**Comment by pjuangph** [comment]
I tried going into my linux laptop manually and checking to see if it's an ssh problem and it's not. The extension actually doesn't work in linux.

**Comment by YiwenZhu77** [comment]
Yes I found the same issue, could the author add support for remote linux system? It will be highly appreciated.
